src/main/java/org/apache/cassandra/service/StorageService.java

```diff
@@ -1812,7 +1812,7 @@ public int scrub(boolean disableSnapshot, String scrubMode, boolean checkData, b
             throws IOException, ExecutionException, InterruptedException {
         MultivaluedMap<String, String> queryParams = new MultivaluedHashMap<String, String>();
         APIClient.set_bool_query_param(queryParams, "disable_snapshot", disableSnapshot);
-        if (scrubMode != "") {
+        if (!scrubMode.isEmpty()) {
```

Comment:
I think `!"".equals(scrubMode)` is safer in case `scrubMode` happens to be a null

Reply:
or check for null explicitly

Reply:
ok, will fix